lbrynet/conf.py

```diff
@@ -591,7 +595,7 @@ def get_conf_filename(self):
         elif os.path.isfile(json_path):
             return json_path
         else:
-            return yml_path
+            return None
```

Comment:
This is redundant here, you can remove it:
```
else:
  return None
```

Reply:
Ok, I got it :) Then I'll remove it as you requested